share/wake/lib/system/path.wake

```diff
@@ -200,3 +200,19 @@ target hashcode (f: String): String =
         match job.isJobOk hash
             True (x, Nil) = x
             _ _ = "BadHash"
+
+# Allow an untracked file to be removed via `wake --clean`
+export target markFileCleanable (filepath: String): Result Unit Error =
```

Comment:
Can this just be implemented as `claim filepath | rmap (\_ Unit)` instead?

Reply:
`claim` might indeed be better, since this method will also trigger "File output by multiple jobs" errors for anyone who *does* try to claim the file, and is not necessarily an easy label to track down.  (By the same token, this won't solve the bug I was running into previously.)

Reply:
Claimed files don't get removed via `wake --clean`

```
$ echo foo > baa
$ wake -x 'claim "baa"'
Pass (Path "baa" "20590a52c4f00588c500328b16d466c982a26fabaa5fa4dcc83052dd0a84f233")
$ wake --clean
$ ls baa
baa
```

Reply:
Ah, that's probably a good thing, now I think about it.